javascript_exercise.js

```diff
@@ -1,5 +1,6 @@
 // The function below is not working properly - can you fix it?
 
-function area(height width) {
+function area(height, width) {
     height * width;
+    console.log(area);
 }
```

Comment:
For this you can write it like this 
```
function area(height, width) {
  return height * width;
}
console.log(area(10, 10));
```
OR
function area(height, width) {
  console.log(height * width);
}
area(10, 10);